Orm/Xtensive.Orm.Manual/Prefetch/PrefetchTest.cs

```diff
@@ -260,14 +259,14 @@ orderby person.Name
           .Prefetch(p => p.Photo) // Lazy load field
           .Prefetch(p => p.Employees // EntitySet Employees
             .Prefetch(e => e.Photo)) // and lazy load field of each of its items
-          .Prefetch(p => p.Manager) // Referenced entity
-          .AsAsync(); 
-        foreach (var person in await prefetchedPersons) {
+          .Prefetch(p => p.Manager); // Referenced entity
+        await foreach (var person in prefetchedPersons.AsAsyncEnumerable()) {

```

Comment:
`ConfigureAwait(false)`

Reply:
There is o need to do this in test code in case we don't synchronously wait for async method completion.